cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -286,6 +289,47 @@ file(
     "${using_mpi}\n"
 )
 
+# handle requested python version if such is specified
+hunter_parse_cmake_args_for_keyword(
+  CMAKE_ARGS "@HUNTER_Boost_CMAKE_ARGS@"
+  KEYWORD REQUESTED_PYTHON_VERSION
+  OUTPUT requested_python
+)
+
+# set up paths for python in boost.user.jam
+if(NOT requested_python STREQUAL "")
+  find_package(PythonInterp ${requested_python} EXACT QUIET)
+  if(PYTHONINTERP_FOUND)
+    set(python_executable ${PYTHON_EXECUTABLE})
+    execute_process(
+      COMMAND "${python_executable}" -c "import os.path, sysconfig; info = sysconfig.get_paths(); print(info['include'])"
+      RESULT_VARIABLE python_process
+      OUTPUT_VARIABLE python_include_directory
```

Comment:
I guess this directory should exist, please add sanity check. The error will mean that we did something wrong or we got unexpected output, so it should be `hunter_internal_error`:
```cmake
if(NOT EXISTS "${python_include_directory}")
  hunter_internal_error("Directory not found: ${python_include_directory}")
endif()
```